source/gaussian_blur.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<p>you can download from here https://github.com/kornia/data/blob/main/bennett_aden.png so that we keep this repo light</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kornia/tutorials/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>